pages/k8s/auth.md

```diff
@@ -289,8 +289,8 @@ in the [upstream documentation][upstream-webhook].
 [docs-ldap]: /kubernetes/docs/ldap
 [roles]: #rbac
 [aws-iam]: /kubernetes/docs/aws-iam-auth
-[auth-webhook-app]: https://github.com/charmed-kubernetes/charm-kubernetes-master/blob/master/templates/cdk.master.auth-webhook.py
-[auth-webhook-svc]: https://github.com/charmed-kubernetes/charm-kubernetes-master/blob/master/templates/cdk.master.auth-webhook.service
+[auth-webhook-app]: https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/blob/master/templates/cdk.master.auth-webhook.py
+[auth-webhook-svc]: https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/blob/master/templates/cdk.master.auth-webhook.service
```

Comment:
@evilnick haven't pages like this moved off to discourse?

Reply:
Yup. There are plenty of broken links in the 1.24 branch. You are welcome to fix them if you like! I usually do that at the end when everything has stopped moving about